src/ai/backend/common/docker.py

```diff
@@ -42,6 +44,18 @@
     "x32": "x86",        # Windows
     "i686": "x86",       # Windows
 }
+# generalize architecture symbols to match docker API's norm
+docker_api_arch_aliases = {
```

Comment:
Let's add type annotation: `Final[Mapping[str, str]]` to indicate it's an immutable constant mapping.